app/views/users/show.html.erb

```diff
@@ -1 +1,12 @@
-<h1>Show user information with the requested id <%= params[:id] %><h1> 
\ No newline at end of file
+<%= stylesheet_link_tag('users_show') %>
+<section class="user_detail">
+    <%= render "user_card", user: @user %>
+    <article class="bio border">
+        <h2>Bio</h2>
+        <p><%= @user.bio %></p>
+    </article>
+    <% @user.posts.each do |post| %>
+    <%= render "posts/post_card", post: post %>
+    <%end%>
+    <button class="see_all">See all posts</button>
```

Comment:
- [ ] From the project requirements, you are expected to run the custom method `recent_posts`  to render 3 recent posts so, please change that 😃 👍